controls/radgridview/row-details/how-to/display-window.md

```diff
@@ -74,12 +75,15 @@ As you can see, it inherits from the [INotifyPropertyChanged](http://msdn.micros
 	
 	private Point CalculateWindowLocation(GridViewRow row)
 	{
-	    throw new NotImplementedException();
+            var lastCell = row.Cells[row.Cells.Count - 1];
+            GeneralTransform generalTransform = lastCell.TransformToVisual(Application.Current.RootVisual);
```

Comment:
I think that the Application.Current.RootVisual property is only available for Silverlight - https://stackoverflow.com/questions/2216917/wpf-equivalent-to-silverlight-rootvisual

We should be able to use Application.Current.MainWindow in WPF.